core/src/main/java/org/elasticsearch/cluster/routing/RoutingTable.java

```diff
@@ -514,12 +517,17 @@ public Builder version(long version) {
             return this;
         }
 
+        /**
+         * Builds the routing table. Note that this can only be called one time.
```

Comment:
I can add something more explicit but calling it more than once will throw an exception in the ImmutableOpenMap. It'd be nice to catch the exception and transform it into something more obvious.